examples/QuickJournal/QuickJournal.Android/QuickJournal.Android.csproj

```diff
@@ -53,10 +53,10 @@
     <Reference Include="System.Numerics.Vectors" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Xamarin.Forms" Version="4.8.0.1451" />
-    <PackageReference Include="Xamarin.Essentials" Version="1.5.3.2" />
+    <PackageReference Include="Xamarin.Forms" Version="5.0.0.2083" />
+    <PackageReference Include="Xamarin.Essentials" Version="1.7.0" />
     <PackageReference Include="Realm">
-      <Version>5.1.1</Version>
+      <Version>10.3.0</Version>
```

Comment:
Would it make sense (if feasible) to just use the surrounding Realm version? User will most likely just check out the whole repo itself right?

Reply:
Do we build and run our examples as part of CI to make sure they always work?

Reply:
I think people look at examples so that they can take inspiration and eventually play with them. If they have to get the whole repo then they'll need to build wrappers and it gets messy, so it's better to have them as separate projects in my opinion. 
Regarding building them, so far it wasn't done. Also examples were not even mentioned at all anywhere

Reply:
Fully agree. They should not need to build wrappers and everything themselves. It should 'just work'.
What I am trying to point at:
We seem to have not updates those examples in while because other tickets were more important. How do we make sure those examples do not get outdated again and keep working with newer versions too?
Someone might just have a look into the examples, copy some code and then it doesn't work with their code because they are using the newest version, not the one the example was built with.

In Cocoa this is part of the test process. We do build all Cocoa examples and if they don't build anymore CI turns red.

Maybe this is out of scope for this ticket and we should create a new one to take care of it. Let me know. :)

Reply:
What you said makes sense, and I think we should address it in a new ticket. Probably it would make sense to update at least the Realm version to every major release we do

Reply:
Would you mind opening a new ticket adding anything that will not be done within this ticket/PR?

Reply:
Sure